hackathon_site/event/test_api.py

```diff
@@ -52,42 +52,77 @@ def test_user_has_profile(self):
         self.assertEqual(response.json(), serializer.data)
 
 
-class CurrentTeamTestCase(SetupUserMixin, APITestCase):
```

Comment:
... so you deleted an entire view's tests? Why?

Reply:
sorry about that